lib/manifest.js

```diff
@@ -34,19 +34,24 @@ module.exports = class Manifest {
 	}
 
 	get(originalPath) {
-		return this._index[originalPath];
+		return this._index.get(originalPath);
 	}
 
 	set(originalPath, actualPath, targetDir) {
 		let key = this.keyTransform(originalPath, targetDir);
 		let uri = this.valueTransform(actualPath);
-		this._index[key] = uri;
+		this._index.set(key, uri);
 
 		let fp = this.filepath;
 		return fp ? createFile(fp, this.toJSON()) : Promise.resolve(null);
 	}
 
 	toJSON() {
-		return JSON.stringify(this._index) + "\n";
+		let index = this._index;
+		let manifest = Array.from(index.keys()).sort().reduce((memo, key) => {
+			memo[key] = index.get(key);
+			return memo;
+		}, {});
```

Comment:
relevant: https://www.stefanjudis.com/today-i-learned/property-order-is-predictable-in-javascript-objects-since-es2015/